include/sleipnir/autodiff/VariableMatrix.hpp

```diff
@@ -19,6 +19,7 @@ namespace sleipnir {
  */
 class SLEIPNIR_DLLEXPORT VariableMatrix {
  public:
+  VariableMatrix();
```

Comment:
Use `VariableMatrix() = default;` instead of defining the impl in the .cpp file.